### DISCUSSION TITLE
specify ":length" and ":pageSize" from variable

### DISCUSSION THREAD

**Discussion by bigdogs** [rocwang/vue-virtual-scroll-grid#132] [Q&A] [answered]
Hi, in my case, the  length and pageSize are from server response after component is mounted, so I can not specify a fix value like demo. 

 I have tried  to set ":length" to "ref(0)", and then change it to real size after server respond, but seems not work, Is there a possible to specify "length" and "pageSize" from from server response.

**Comment by rocwang** [comment, marked as answer]
I think `vue-virtual-scroll-grid` can support such feature, but not sure if I can find some spare time to implement it in the near future though. 😅 (PR is welcome.)

For now, as a workaround, you might consider setting the props to a default value (e.g length: 10, page: 10), then when you get the actual `length` and `pageSize` from the server, then you update the props. Will this work for you?

**Reply by bigdogs** [reply]
@rocwang Set length to non zero value seems work ~ Thanks for your support :)
